extensions/wikia/Discussions/Discussions.setup.php

```diff
@@ -66,3 +66,5 @@
 $wgGroupPermissions['staff']['specialdiscussions'] = true;
 
 $wgHooks['WikiaSkinTopScripts'][] = 'DiscussionsHooksHelper::addDiscussionJsVariable';
+
+$wgDiscussionsApiUrl = $wgServicesExternalDomain . 'discussion';
```

Comment:
Should this one use the URL provider wherever it is used as well?

Reply:
fix: 3c073840caa